### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akfernun**
Provide more documentation on new features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,18 +91,32 @@ After that you have the following instance methods available to you.
...
+    `TelephoneNumber.valid?("3175082237", :US, [:mobile, :fixed_line]) ==> true`
+    `TelephoneNumber.valid?("3175082237", :US, [:toll_free]) ==> false`
+    
   - #### `invalid?`
   
     Returns boolean value indicating whether or not a particular number is invalid.
     
     `TelephoneNumber.invalid?("3175082237", :US) ==> false`
     
+    If you are looking to validate against a specific set of keys, you can pass in an array of keys
```
Comment (trouta23) [inline README.md]: Should this be invalidate?

File: README.md [hunk 2]
```diff
@@ -91,18 +91,32 @@ After that you have the following instance methods available to you.
...
+    
   - #### `valid?`
     
     Returns boolean value indicating whether or not a particular number is valid.
     
     `TelephoneNumber.valid?("3175082237", :US) ==> true`
     
+    If you are looking to validate against a specific set of keys, you can pass in an array of keys
+    
+    `TelephoneNumber.valid?("3175082237", :US, [:mobile, :fixed_line]) ==> true`
```
Comment (trouta23) [inline README.md]: The formatting looks weird here, but that's more of a nitpick than anything:

mobi_telephone_number_at_akfernun-patch-1

File: README.md [hunk 3]
```diff
@@ -91,18 +91,32 @@ After that you have the following instance methods available to you.
...
+    
   - #### `invalid?`
   
     Returns boolean value indicating whether or not a particular number is invalid.
     
     `TelephoneNumber.invalid?("3175082237", :US) ==> false`
     
+    If you are looking to validate against a specific set of keys, you can pass in an array of keys
+    
+    `TelephoneNumber.invalid?("3175082237", :US, [:mobile, :fixed_line]) ==> false`
```
Comment (trouta23) [inline README.md]: Same here

File: README.md (hunk 2 above)
Comment (akfernun) [inline README.md]: gahhh thanks for catching that. I could've sworn it didn't render like this when I edited it.

File: README.md (hunk 1 above)
Comment (akfernun) [inline README.md]: "Invalidate" really is a word

**Pull request merged**